tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -239,24 +238,6 @@ describe('Boundary Timer Event', () => {
               .should($elements => {
                 expect($elements).to.have.lengthOf(numberOfBoundaryTimerEventsAdded);
               });
-
-            moveElement(taskPosition, 200, 400);
-            waitToRenderAllShapes();
-            cy.wrap($boundaryEvent).should($el => {
-              const { left, top } = $el.position();
-              const positionErrorMargin = 2;
-              expect(left).to.be.closeTo(initialBoundaryEventPosition.left, positionErrorMargin);
-              expect(top).to.be.closeTo(initialBoundaryEventPosition.top, positionErrorMargin);
-            });
-
-            moveElement(task2Position, 300, 500);
-            waitToRenderAllShapes();
-            cy.wrap($boundaryEvent).should($el => {
-              const { left, top } = $el.position();
-              const positionErrorMargin = 2;
-              expect(left).to.not.be.closeTo(initialBoundaryEventPosition.left, positionErrorMargin);
-              expect(top).to.not.be.closeTo(initialBoundaryEventPosition.top, positionErrorMargin);
-            });
```

Comment:
Why where these tests removed? I added them back and they appear to be failing; I don't think the solution to failing tests is to remove them altogether.

It appears this is suppose to be testing that boundary events snap back to their original position, and that moving the original task doesn't change the position of the boundary event which should now be attached to the other task.

It seems the first part of this ("snaps back to original position") may already captured in another test, but not the other part. If this test is unclear, maybe it can be moved into a separate test. What do you think?

Reply:
Lisa and I discussed this. The test was testing two things and I suggested removing this. I think Lisa noted this and added a task to add a separate (common?) test for this case. The failure also seems to be related to some potentially buggy behaviour around how labels are hidden/re-shown.

Reply:
I added a task to at least add the label back after dropping a boundary event from one task to another. #723 
That might be related to why the left position is so different, might not.

I can add another task to re-add the test for "if the boundary event is in the expected location after dragging a task".

Reply:
For whatever reason the $boundaryEvent.position() and $el.position() left position is off about 62 pixels. But manually it is working fine.

Reply:
If I click on a task after the boundary event was moved and then store the event position - it looks like I can get the second part of the test to work. I'm not as familiar with the cy.wrap so I was trying to click with the wrong functions before. I could make an issue to split out those tests, or do it now.